File: benchmark/fibonacci_recursive/fib.go
package main

import "fmt"

func fib(n int64) int64 {
	if n < 2 {
		return n
	}
	return fib(n-1) + fib(n-2)
}

func main() {
	var n int64 = 40
	result := fib(n)
	fmt.Printf("Fib(%d) = %d\n", n, result)

}

File: benchmark/fibonacci_recursive/fib.js
function fib(n) {
  if (n < 2) return n;
  return fib(n - 1) + fib(n - 2);
}

function main() {
  const n = 40;
  const result = fib(n);
  console.log(`Fib(${n}) = ${result}`);
}

main();
